lib/usage_data.js

```diff
@@ -10,12 +10,13 @@ const { v4: uuidv4 } = require('uuid')
 
 // Local dependencies
 const packageJson = require('../package.json')
+const { projectDir } = require('./utils')
 
 exports.getUsageDataConfig = function () {
   // Try to read config file to see if usage data is opted in
   let usageDataConfig = {}
   try {
-    usageDataConfig = require(path.join(__dirname, '../usage-data-config.json'))
+    usageDataConfig = require(path.resolve('usage-data-config.json'))
```

Comment:
The get and set are using different paths. Could we make a path variable for them to share please?

(this could lead to the user being asked, then the answer not being available so them being asked again each time they start)

Reply:
The example would be:

`const usageDataFilePath = //whatever`

`usageDataConfig = require(usageDataFilePath)`

`fs.writeFileSync(usageDataFilePath, usageDataConfigJSON)`